cda.py

```diff
@@ -102,19 +105,51 @@ def main_snapshot(args: argparse.Namespace) -> None:
     else:
         logger.error(f"Unexpected action {args.actions}")
 
+def main_post_config(args) -> None: 
```

Comment:
I think it makes more sense to treat running specific post config steps as part of our steps argument, otherwise you can't skip specific post config steps.

Currently the user can run:

`--steps pre,masters,workers,post`

We could extend it so that he can run:
`--steps pre,masters,workers,post(hwol_offload,...)`
By default, it runs all pre and post if the user doesn't specify anything.